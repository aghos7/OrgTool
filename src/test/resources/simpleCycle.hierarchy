1, 1, Org1Cycle