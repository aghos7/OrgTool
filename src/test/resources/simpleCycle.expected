Nothing, it should throw an exception that it found a cycle